app/templates/index.hbs

```diff
@@ -1 +1,4 @@
-Index page works!
\ No newline at end of file
+<IndexButtons 
```

Comment:
Does this name limit us from creating a reusable component in future?

Reply:
Sorry I could not think of any other name at that time! Could I use 'UserStatus' ?

Reply:
What status of the user does it tell us?